provisioning/device/src/ProvisioningDeviceClient.cs

```diff
@@ -37,27 +37,18 @@ public class ProvisioningDeviceClient
                 CertificateInstaller.EnsureChainIsInstalled(x509Auth.GetAuthenticationCertificateChain());
             }
 
-            if (options == default)
-            {
-                options = new();
-            }
+            _options = options != default
+                ? options.Clone()
+                : new ();
```

Comment:
Seems the style is no space between new and parens.

```suggestion
                : new();
```